agent/agent.go

```
@@ -652,8 +654,11 @@ func (r *agent) collectLocalStatus(ctx context.Context, local *serf.Member) (sta
 }
 
 // getLocalStatus obtains local node status.
-func (r *agent) getLocalStatus(ctx context.Context, local serf.Member, respc chan<- *statusResponse) {
-	status, err := r.collectLocalStatus(ctx, &local)
+func (r *agent) getLocalStatus(ctx context.Context, respc chan<- *statusResponse) {
+	// TODO: restructure code so that local member is not needed here.
+	local, _ := r.ClusterMembership.FindMember(r.Name)
```

Do not ignore the error.